frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -63,7 +63,8 @@ class BrfssProvider extends VariableProvider {
       !breakdowns.time &&
       (breakdowns.geography === "state" ||
         breakdowns.geography === "national") &&
-      (!breakdowns.demographic || breakdowns.demographic === "race")
+      breakdowns.demographicBreakdownCount() === 1 &&
+      !!breakdowns.race
```

Comment:
Previously this allowed no demographic breakdown - don't you want both `0` and `1` to be valid counts?
```
breakdowns.demographicBreakdownCount() === 0 || (breakdowns.demographicBreakdownCount() === 1 && !!breakdowns.race)
```